index.html

```diff
@@ -282,6 +282,46 @@ <h3>Skills</h3>
         </div>
     </div>
     </section>
+    <!-- From starting from here -->
+    <section class="background-cont">
+      <form method="post" action="https://formspree.io/f/mrgdozpa" class="Contact-me">
+        <legend>Contact me</legend>
+        <p class="">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <label for="">
+        <input
+          type="text"
+          placeholder="Type Name"
+          name="Name"
+          class="cont-input"
+          id="name"
+          value="Yeremias"
```

Comment:
- [Optional] I think the placeholder for the input name and email is enough. kindly remove it. by removing the value you let the viewer to contact you easily.